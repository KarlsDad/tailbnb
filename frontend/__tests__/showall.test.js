import React from 'react';
import { shallow } from 'enzyme';

// Component
import ShowAll from '../src/components/ShowAll';

describe('ShowAll', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<ShowAll />);
    expect(component).toMatchSnapshot();
  });
});
