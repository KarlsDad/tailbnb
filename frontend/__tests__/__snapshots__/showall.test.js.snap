// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShowAll should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
